infra/src/main/java/com/redhat/service/smartevents/infra/processor/actions/KafkaTopicConstants.java

```diff
@@ -0,0 +1,10 @@
+package com.redhat.service.smartevents.infra.processor.actions;
+
+public class KafkaTopicConstants {
+    public static String TOPIC_PARAM = "topic";
+    public static String KAFKA_BROKER_URL = "kafka_broker_url";
+    public static String KAFKA_CLIENT_ID = "kafka_client_id";
+    public static String KAFKA_CLIENT_SECRET = "kafka_client_secret";
+    public static String KAFKA_SECURITY_PROTOCOL = "kafka_security_protocol";
+    public static String BRIDGE_ERROR_TOPIC_NAME = "bridge_error_topic_name";
+}
```

Comment:
No needed anymore?

Reply:
it's needed as the key needs to be in both processor/configuration and DTO (which is in infra)

Reply:
Removed